### PR TITLE
fifo animal shelter

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/fifoAnimalShelter/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
++ Create a class called AnimalShelter that holds dogs and cats. With a first in, first-out approach
+
+## API, Approach & Efficiency
+
++ enqueue(animal) - O(1) - Adds animals to the shelter either cats or dogs
++ dequeue(pref) - O(n) - Returns either dogs or cats and if neither returns null
+
+### Whiteboard
+
++ ![Fifo Animal Shelter](fifoAnimalShelter1.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/fifoAnimalShelter/README.md:18]: Whiteboard and code is looking really great! Would just like to see some tests added!
